File: IDAUtil/Model/CustomerDataProperty.vb
Imports System.ComponentModel.DataAnnotations.Schema

Public Class CustomerDataProperty
    <Column("[salesOrg]")>
    Public Property salesOrg As String
    <Column("[distributionChannel]")>
    Public Property distributionChannel As Long
    <Column("[country]")>
    Public Property country As String
    <Column("[soldTo]")>
    Public Property soldTo As Long
    <Column("[soldToName]")>
    Public Property soldToName As String
    <Column("[shipTo]")>
    Public Property shipTo As Long
    <Column("[shipToName]")>
    Public Property shipToName As String
    <Column("[cmirCheckAllowed]")>
    Public Property cmirCheckAllowed As Boolean
    <Column("[replaceObsoletePartialCutsAllowed]")>
    Public Property replaceObsoletePartialCutsAllowed As Boolean
    <Column("[sendOrderConfirmationAllowed]")>
    Public Property sendOrderConfirmationAllowed As Boolean
    <Column("[sendDeliveryConfirmationAllowed]")>
    Public Property sendDeliveryConfirmationAllowed As Boolean
    <Column("[changeRouteCodeActionAllowed]")>
    Public Property changeRouteCodeActionAllowed As Boolean
    <Column("[changeRDDActionAllowed]")>
    Public Property changeRDDActionAllowed As Boolean
    <Column("[oneDayLeadTimeAllowed]")>
    Public Property oneDayLeadTimeAllowed As Boolean
    <Column("[leadTime]")>
    Public Property leadTime As Integer
    <Column("[deliveryDay]")>
    Public Property deliveryDay As String
    <Column("[sendDailyCutsAllowed]")>
    Public Property sendDailyCutsAllowed As Boolean
    <Column("[accountManager]")>
    Public Property accountManager As String
    <Column("[salesDistrict]")>
    Public Property salesDistrict As Long
    <Column("[csrNumber]")>
    Public Property csrNumber As Long
    <Column("[csrName]")>
    Public Property csrName As String
    <Column("[minimumOrderCaseQuantity]")>
    Public Property minimumOrderCaseQuantity As Long
    <Column("[minimumOrderPalletQuantity]")>
    Public Property minimumOrderPalletQuantity As Long
    <Column("[minimumOrderValue]")>
    Public Property minimumOrderValue As Long
    <Column("[truckOverflowPalletQuantity]")>
    Public Property truckOverflowPalletQuantity As Long
    <Column("[truckOverflowValue]")>
    Public Property truckOverflowValue As Long
    <Column("[truckOverflowWeight]")>
    Public Property truckOverflowWeight As Long
    <Column("[markedForDelitionZA01Only]")>
    Public Property markedForDelitionZA01Only As Boolean

End Class

File: IDAUtil/Model/Properties/ServerProperty/CustomerDataProperty.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace IDAUtil.Model.Properties.ServerProperty { 
    public class CustomerDataProperty {
        [Column("[salesOrg]")] public string salesOrg { get; set; }
        [Column("[distributionChannel]")] public int distributionChannel { get; set; }
        [Column("[country]")] public string country { get; set; }
        [Column("[soldTo]")] public int soldTo { get; set; }
        [Column("[soldToName]")] public string soldToName { get; set; }
        [Column("[shipTo]")] public int shipTo { get; set; }
        [Column("[shipToName]")] public string shipToName { get; set; }
        [Column("[cmirCheckAllowed]")] public bool cmirCheckAllowed { get; set; }
        [Column("[replaceObsoletePartialCutsAllowed]")] public bool replaceObsoletePartialCutsAllowed { get; set; }
        [Column("[changeRouteCodeActionAllowed]")] public bool changeRouteCodeActionAllowed { get; set; }
        [Column("[changeRDDActionAllowed]")] public bool changeRDDActionAllowed { get; set; }
        [Column("[oneDayLeadTimeAllowed]")] public bool oneDayLeadTimeAllowed { get; set; }
        [Column("[leadTime]")] public int leadTime { get; set; }
        [Column("[deliveryDay]")] public string deliveryDay { get; set; }
        [Column("[accountManager]")] public string accountManager { get; set; }
        [Column("[salesDistrict]")] public int salesDistrict { get; set; }
        [Column("[csrNumber]")] public int csrNumber { get; set; }
        [Column("[csrName]")] public string csrName { get; set; }
        [Column("[minimumOrderCaseQuantity]")] public int minimumOrderCaseQuantity { get; set; }
        [Column("[minimumOrderValue]")] public int minimumOrderValue { get; set; }
        [Column("[truckOverflowPalletQuantity]")] public int truckOverflowPalletQuantity { get; set; }
        [Column("[truckOverflowValue]")] public int truckOverflowValue { get; set; }
        [Column("[truckOverflowWeight]")] public int truckOverflowWeight { get; set; }
        [Column("[markedForDelitionZA01Only]")] public bool markedForDelitionZA01Only { get; set; }
        [Column("[region]")] public string region { get; set; }
        [Column("[orderConfirmationEmails]")] public string orderConfirmationEmails { get; set; }
        [Column("[deliveryConfirmationEmails]")] public string deliveryConfirmationEmails { get; set; }
        [Column("[dailyCutsEmails]")] public string dailyCutsEmails { get; set; }
        [Column("[belowMOQandMOVEmails]")] public string belowMOQandMOVEmails { get; set; }
    }
}